cordapp/src/main/kotlin/com/template/App.kt

```diff
@@ -33,30 +32,26 @@ class TemplateApi(val rpcOps: CordaRPCOps) {
 @StartableByRPC
 class Initiator : FlowLogic<Unit>() {
     @Suspendable
-    override fun call() {
-        return Unit
-    }
+    override fun call() = Unit
```

Comment:
This makes sense, but we chose to leave in the curly braces because we were worried newcomers would be confused how to extend the the function to have multiple statements if you use `=`.

Reply:
How about

```kotlin
@Suspendable
    override fun call() {
        // Flow implementation goes here    
    }
```

(`return Unit` is not required in this context)

Reply:
Sounds good :)